### ISSUE TITLE
Articulate how sensorML Sensor models and parameters relate to SSN

### ISSUE THREAD

**Issue by rob-metalinkage** [w3c/sdw#1125]
O&M has a concept observation procedures, and SensorML codifies a model for this :

"the procedure property within an Observation instance may reference a SensorML description of the measurement process" 

also

"As discussed in the Core Concepts, any process can have inputs, outputs, and parameters"

SSN does not make any reference to such "parameters" although some concepts such as capability are explicitly modelled. 

is this out of scope, an oversight, or a valid concern for an extension?

**Comment by lieberjosh** [comment]
The latter, I believe. A SensorML extension for SSN would be quite valuable.

—Josh

> Josh Lieberman, Ph.D.
> jlieberman@opengeospatial.org
> Tel. +1 617 431 6431

**Comment by dr-shorthair** [comment]
Note that SSN split O&M's `om:procedure` into 
- `sosa:usedProcedure : sosa:Procedure` - which is the reusable recipe or protocol 
- `sosa:madeBySensor : sosa:Sensor` - the (usually) hardware that implements the procedure

SensorML's single document with optional pieces conflates the type and the instance.

**Comment by rob-metalinkage** [comment]
thanks. In the case of "SensorModel" this seems to suggest:

sosa:Sensor should define the model and its parameters - via sensor types which are subclassed from sosa:Sensor?

sosa:Procedure should have values for the sensor parameters

(SOSA is flexible whether sensors are types or instances - and i didnt identify the pattern where we want to declare the type _and_ the instance so we can find the type of the instance - so perhaps subclassing of an instance is the only way to do this?

**Comment by dr-shorthair** [comment]
https://www.w3.org/TR/vocab-ssn/#SOSASensor - Device, agent (including humans), or software (simulation) involved in, or implementing, a Procedure. Sensors respond to a Stimulus, e.g., a change in the environment, or Input data composed from the Results of prior Observations, and generate a Result. Sensors can be hosted by Platforms. ... 

https://www.w3.org/TR/vocab-ssn/#SOSAProcedure - A workflow, protocol, plan, algorithm, or computational method specifying how to make an Observation, create a Sample, or make a change to the state of the world (via an Actuator). ...

**Comment by rob-metalinkage** [comment]
yep - but that doesnt actually make it clear whether a sensor _model_ is attached to a _type_ of sensor or a procedure, and SSN doesnt currently provide the properties and classes for a sensor model and its parameters... hence the original question - seems like a useful extension.

**Comment by ghobona** [comment]
Within the context of imagery, a sensor model is defined as a "geopositioning mathematical description of the relationship between the three-dimensional object space and the two-dimensional plane of the associated image produced by a sensor" (ISO 19130-1). So perhaps what is needed is an extension of the class https://www.w3.org/TR/vocab-ssn/#SOSAProcedure
 
The definition of SOSAProcedure appears to support this, given that it refers to an "algorithm, or computational method specifying how to make an Observation".
